src/databricks/labs/ucx/cli.py

```diff
@@ -282,6 +282,24 @@ def principal_prefix_access(w: WorkspaceClient, ctx: WorkspaceContext | None = N
     raise ValueError("Unsupported cloud provider")
 
 
+@ucx.command
+def create_missing_principals(
+    w: WorkspaceClient,
+    prompts: Prompts,
+    ctx: WorkspaceContext | None = None,
+    single_role: bool = False,
+    **named_parameters,
+):
+    """No supported for Azure.
+    For AWS, this command identifies all the S3 locations that are missing a UC compatible role and creates them.
+    It takes single_role optional parameter. If set to True, it will create a single role for all the S3 locations."""
```

Comment:
```suggestion
By default, it will create 1 role per S3 location. Specify the optional single_role parameter to True, to create a single role for all the S3 locations."""
```